Extension/src/background/services/locale-detect.ts

```diff
@@ -100,10 +137,45 @@ export class LocaleDetect {
         'il': 'he',
         // Chinese
         'cn': 'zh',
+        'tw': 'zh',
         // Indonesian
         'id': 'id',
+        // Malaysian
+        'my': 'id',
         // Turkish
         'tr': 'tr',
+        // Serbian
+        'sr': 'sr',
+        'ba': 'sr',
+        // Croatian
+        'hr': 'hr',
+        // Hindi
+        'in': 'hi',
+        // Bangla:
+        'bd': 'hi',
+        // Sri Lanka
+        'lk': 'hi',
+        // Nepal:
+        'np': 'hi',
+        // Estonian:
+        'ee': 'et',
+        // Persian:
+        'ir': 'fa',
+        // Tajik:
+        'tj': 'fa',
+        // Korean:
+        'kr': 'ko',
+        // Danish:
+        'dk': 'da',
+        // Faroese:
+        'fo': 'fo',
+        // Vietnamese:
+        'vn': 'vi',
+        // Thai:
+        'th', 'th',
+        // Swedish:
+        'se', 'sv',
+        'ax', 'sv',
```

Comment:
Looks like a typo :) should be `:` instead of `,`

Reply:
Should be correct too now.